docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3'
```

Comment:
Check if this can be removed, as I get the following output;
```bash
$ docker-compose build
WARN[0000] /home/lakshaya/RAndD/node-server-docker-sample/docker-compose.yml: `version` is obsolete
```